### ISSUE TITLE
Marker accessibility

### ISSUE THREAD

**Issue by Malvoz** [mapbox/mapbox-gl-js#11763]
How is marker accessibility achieved in MapBox?

- Should [default markers](https://docs.mapbox.com/mapbox-gl-js/example/add-a-marker/) be focusable?
- Is there a need for authoring guidance? Similar to Leaflet's [Guideline for accessible markers](https://leafletjs.com/SlavaUkraini/examples/accessibility/#markers-must-be-labelled).
- [Autopan markers into view on focus](https://github.com/Leaflet/Leaflet/issues/8040) is great for keyboard users, please consider it.

**Comment by vikraman-alea-bah** [comment]
This isn't really a feature request, it's an accessibility bug. There's a class action lawsuit against Hyatt right now addressing this very issue with another map package. If markers are not focusable  for screen readers and keyboard only users, it opens up everyone using this to a lawsuit. 

Since this post is a few years old, is there any new documentation on how to address this or does anyone know if this bug is being addressed?

**Comment by vikraman-alea-bah** [comment]
Does anyone know if there has been any movement on this accessibility bug?
